src/js/Framework/FormCollection.js

```diff
@@ -26,6 +29,11 @@ const FormCollection = function (element) {
       dragClass: 'collection-item-selected'
     })
   }
+
+  // Add minimum items
+  if (this.numberOfItems < this.minimumItems) {
```

Comment:
Does this not have to be a foreach? If minimum items is 3, only 1 item will be added?

Reply:
fixed f6afd9f7b1a1c625450d35bea98d6daffcbff6d1